app/views/katello/api/v2/subscriptions/show.json.rabl

```diff
@@ -59,10 +59,3 @@ node :activation_keys do |subscription|
     }
   end
 end
-
-node :host, :if => lambda { |sub| sub && sub.host } do |subscription|
```

Comment:
@johnpmitsch actually i might remove it from base and put it back in show to help with performance on index, sound good?